django_slack/api.py

```diff
@@ -132,6 +137,9 @@ def slack_message(template, context=None, attachments=None, fail_silently=None,
 
         if 'attachments' in data:
             data['attachments'] = json.dumps(data['attachments'])
+        
+        if 'blocks' in data:
```

Comment:
Neat, now we can have a nice loop over a `('attachments', 'blocks')` tuple so do avoid some duplicated code here.